modules/search/class-jetpack-instant-search.php

```diff
@@ -132,10 +132,7 @@ protected function inject_javascript_options() {
 			'postsPerPage'          => get_option( 'posts_per_page' ),
 			'siteId'                => Jetpack::get_option( 'id' ),
 
-			// filtering.
-			'postTypeFilters'       => isset( $widget_options['post_types'] ) ? $widget_options['post_types'] : array(),
 			'postTypes'             => $post_type_labels,
-			'sort'                  => isset( $widget_options['sort'] ) ? $widget_options['sort'] : null,
```

Comment:
Is this deletion intentional? If so, we also need to remove handling for `initialSort`:

https://github.com/Automattic/jetpack/blob/add/search-filter-query/modules/search/instant-search/index.jsx#L30

https://github.com/Automattic/jetpack/blob/add/search-filter-query/modules/search/instant-search/components/search-app.jsx#L54

Reply:
I thought we were planning to hide any sorting options from the widget customization anyways? The widget sorting option seems confusing since you could have different settings in different places.

Reply:
In that case, I think it'd be best to remove handling for `ininitalSort` in a future PR.